tools/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+add_executable(sof-fuzzer
+	main.c
+	qemu-bridge.c
+	topology.c
+	platform/byt-host.c
+)
+
+target_link_libraries(sof-fuzzer PRIVATE
+	"-lglib-2.0 -lrt"
+)
+
+target_include_directories(sof-fuzzer PRIVATE
+	"/usr/include/glib-2.0"
```

Comment:
Hardcoding is not the way to go. In cmake you find packages, it's not something that will differ on win vs linux, it will also differ between linux distros. You should try something like this:
```
find_package(PkgConfig REQUIRED)
pkg_search_module(GLIB REQUIRED glib-2.0)
target_include_directories(mytarget PRIVATE ${GLIB_INCLUDE_DIRS})
target_link_libraries(mytarget INTERFACE ${GLIB_LDFLAGS})
```

Reply:
Thanks @jajanusz let me give this a try.